2-Security/1-intro.md

```diff
@@ -0,0 +1,40 @@
+## Security
+
+Now that you (probably) know some stuff about the internet, it's valuable to
+learn how you can protect yourself from people who want to attack you and your
+services.
+
+A long time ago, people used protocols across local networks to access files.
+Assuming no one could intercept traffic, there was no reason to worry about
+whether or not someone could replace the files with a malicious verson.
+Obviously, this is no longer the case, and more and more cases continue to rise
+up about how people end up getting hacked and your personal data ends up thrown
+out onto the internet.
+
+Software nowadays goes through a very long process to make it from the entity
```

Comment:
This one reads a bit funny.

Maybe?
```suggestion
Software nowadays goes through a very long process to from the entity who made the software to your computer before you can use it.
```

Reply:
remove first "to"

Reply:
Your suggested fix doesn't read right. I reworded it but "to from the entity" doesn't read right at all.

Reply:
The rewording makes it much nicer. :+1: